src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/MarkdownPreviewSingleton.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Markdown.Platform;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class MarkdownPreviewSingleton
+    {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Can we put these in the global overrides so we don't have to wrap every call in these tags?

Reply:
I don't get the option to in VS.

Reply:
I also have to make changes to this class when MarkdownPreview is updated so I can try again in that PR.